lms/djangoapps/commerce/tests/test_utils.py

```diff
@@ -102,6 +102,13 @@ def test_get_receipt_page_url(self):
         expected_url = 'http://ecommerce_url/checkout/receipt/?order_number={}'.format(order_number)
         self.assertEqual(url, expected_url)
 
+    @override_settings(ECOMMERCE_PUBLIC_URL_ROOT='http://ecommerce_url')
+    def test_get_order_dashboard_url(self):
+        """Verify that the proper order dashboard url is returned."""
+        url = EcommerceService().get_order_dashboard_url()
+        expected_url = 'http://ecommerce_url/dashboard/orders'
```

Comment:
it's `/dashboard/orders/` in the variable above - just checking: should it be `/orders` here or `/orders/`?